OpenRA.Mods.Common/Traits/Attack/AttackFrontal.cs

```diff
@@ -32,27 +33,20 @@ public override void RulesetLoaded(Ruleset rules, ActorInfo ai)
 
 	public class AttackFrontal : AttackBase
 	{
-		readonly AttackFrontalInfo info;
+		public readonly AttackFrontalInfo FrontalInfo;
```

Comment:
Would it make more sense consistency-wise to use `public new readonly AttackFrontalInfo Info` instead?

Reply:
Yes, Fixed.